.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - yarn/setup
       - run:
           name: Danger
-          command: yarn danger ci
+          command: DANGER_GITHUB_API_TOKEN="15e52de81a772b174cc5""e1813d0083564c69c325" yarn danger ci
```

Comment:
Why did the api key need to be added, and does this expose anything in terms of force being OSS?

Reply:
I'm not sure! @orta had me add that after seeing the build failures.

Reply:
PRs on Forks don't get the access token, so danger doesn't run